src/lib/ng-package/entry-point/entry-point.spec.ts

```diff
@@ -33,7 +32,8 @@ describe(`NgEntryPoint`, () => {
     // Test readonly getters:
     expect(primaryEntryPoint.entryFilePath).to.equal(resolve(testSourcePath, 'public-api.ts'));
     expect(primaryEntryPoint.isSecondaryEntryPoint).to.be.false;
-    expect(primaryEntryPoint.destinationPath).to.equal(resolve(testDestinationPath));
+    expect(primaryEntryPoint.entryPointDestinationPath).to.equal(resolve(testDestinationPath));
+    expect(primaryEntryPoint.libraryDestinationPath).to.equal(resolve(testDestinationPath));
```

Comment:
Is `NgEntryPoint.destinationPath` a public API? Does it need to go through a deprecation phase?

Reply:
Strictly speaking the changes here are part of the public API. While this shouldn't effect anyone. I think It would be better to leave such changes for a major version. (v12), as we definitely shouldn't break anyone in a minor even though this API is not stable at all.

Reply:
Ok, I'll rename these so that we don't have a breaking change. And I'll add a TODO so that we have better names in an upcoming major version.

Thank!

Reply:
Would there still be a breaking change because of the change of behaviour?  Or, maybe I am not understanding what you mean by renaming.

Reply:
I'll add back the old `NgEntryPoint.destinationPath`, mark it as deprecated and  I will make sure that it works the same way, after all, it's just a getter.

Thanks for the quick reply.

Reply:
`NgEntryPoint.destinationPath` is back and it's no longer deprecated. And I've also reduced the amount of changes made in this PR.